docs/docs/documentation/getting-started/quick-start.md

```diff
@@ -45,7 +82,7 @@ solace-agent-mesh init --gui
 You can also initialize your Solace Agent Mesh project using the official Docker image. This is helpful if you want to avoid local Python/SAM CLI installation or prefer a containerized workflow from the start.
 
 ```sh
-docker run --rm -it -v "$(pwd):/app" -p 5002:5002 solace/solace-agent-mesh:latest init --gui
+docker run --rm -it -v "$(pwd):/app" --platform linux/amd64  -p 5002:5002 solace/solace-agent-mesh:latest init --gui
 ```
 
 If the OS architecture on your host is not `linux/amd64`, you would need to add `--platform linux/amd64` when running container.
```

Comment:
If the OS architecture on your host is not `linux/amd64`, you must add `--platform linux/amd64` when you run the container.